dev-tools/ci-integration/step1-changed_files.sh

```diff
@@ -0,0 +1 @@
+IFS=$'\n'; CHANGED_FILES=($(git diff --name-only --diff-filter=ACMRTUXB "${COMMIT_RANGE}")); unset IFS
```

Comment:
right
artifact from `bats` approach when it requested each testing command to be separated file